Assets/_BForBoss/_Core/Scripts/Environment/EnemySpawnAreaBehaviour.cs

```diff
@@ -12,6 +15,8 @@ public class EnemySpawnAreaBehaviour : MonoBehaviour
         [SerializeField] [Min(0.0f)] private float _secondsBetweenSpawns = 2.5f;
         [SerializeField, Tooltip("Should max amount of enemies spawn upon initialization")]
         private bool _burstInitialSpawn = true;
+        [SerializeField] 
+        private float _spawnRadius = 1;
```

Comment:
maybe `_spawnRadiusMultiplier`? and to ensure no one puts a crazy value, you can define a range (i.e. between 0 and 3 maybe??) - at least add a Min attribute to ensure no negative values are used.

Also think a Tooltip would be really useful

Reply:
It's not a multiplier, it's literally just the size of the box. it's not multiplying anything.